docs/source/installing-server.md

```diff
@@ -122,41 +122,41 @@ $ git clone git@github.com:bigchaindb/bigchaindb.git
 
 Build the Docker image:
 ```text
-$ docker-compose build
+$ docker build --tag local-bigchaindb .
 ```
 
-then do a one-time configuration step to create the config file; it will be
-stored on your host machine under ` ~/.bigchaindb_docker/config`:
+then do a one-time configuration step to create the config file; we will use
+the `-y` option to accept all the default values. The configuration file will
+be stored on your host machine under ` ~/bigchaindb_docker/.bigchaindb`:
 ```text
-$ docker-compose run --rm bigchaindb bigchaindb configure
-Starting bigchaindb_rethinkdb-data_1
+$ docker run --rm -v "$HOME/bigchaindb_docker:/data" -ti local-bigchaindb -y configure
 Generating keypair
-API Server bind? (default `localhost:9984`): 
-Database host? (default `localhost`): rethinkdb
-Database port? (default `28015`): 
-Database name? (default `bigchain`): 
-Statsd host? (default `localhost`): statsd
-Statsd port? (default `8125`): 
-Statsd rate? (default `0.01`): 
+Configuration written to /data/.bigchaindb
 Ready to go!
 ```
 
-As shown above, make sure that you set the database and statsd hosts to their
-corresponding service names (`rethinkdb`, `statsd`), defined in`docker-compose.yml`
-and `docker-compose-monitor.yml`.
-
 You can then start it up (in the background, as a daemon) using:
 ```text
-$ docker-compose up -d
+$ BIGCHAIN_CID=$(docker run -d -v "$HOME/bigchaindb_docker:/data" local-bigchaindb)
```

Comment:
looks nice but I think it's only for `docker-compose`, will dig more later to see if the option is on `docker` itself.